tutorial/tutorial-gabc-01.html

```diff
@@ -89,7 +89,7 @@
 </pre>
 </div>
 
-<p>Here's the result after processing by gregorio and LaTeX:</p>
+<p>Here's the result after processing by gregorio and <span class="latex">L<span class="alpha">a</span>T<span class="epsilon">e</span>X</span>:</p>
```

Comment:
In the steps that follow, it seems that new/changed notes are highlighted in red.  Should this not be the case for `(c4)` and `(f)` here?